src/main/groovy/CustomGet.groovy

```diff
@@ -40,12 +48,12 @@ class CustomGet {
                 identifier: "x.url",
                 fields:
                         [
-                                ["name": "url", "displayName": "URL", "fieldType": "text", "i18n":"URL"],
+                                ["name": "url", "displayName": "URL", "fieldType": "text", "i18n":"URL", "extended_type":"url"],
                         ],
                 screens:
                         [
                                 [
-                                        header: "Enter the URL that you wish to pull content from",
+                                        header: "Availability check for URL",
```

Comment:
This is going to be confusing because this CustomGet recipe does not include an availability check response like the new CustomGetAvailability recipe.   This recipe just grabs whatever data the customer puts at the endpoint and returns it so the original header is better.